__tests__/api.test.js

```diff
@@ -0,0 +1,43 @@
+const app = require('../app')
+const request = require('supertest')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+
+
+beforeEach(() => seed(testData))
+
+afterAll(() => db.end())
+
+describe("News API BACKEND PROJECT", () => {
+    describe("GET /api/topics that returns all topics", () => {
+
+        test('returns a 200 response status', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+        })
+        test('returns a 200 response status and data with correct properties', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                const { body } = response
+                expect(Array.isArray(body)).toBe(true)
+                body.forEach((topic) => {
+                    expect(topic).toHaveProperty("slug")
+                    expect(topic).toHaveProperty("description")
+                })
+            })
+        })
```

Comment:
Really nice testing for the /api/topics endpoint. I like the idea of the first test, but you are expecting a status code of 200 in the second test as well making the first one redundant. I would recommend either removing the first test entirely, or removing the `.expect(200)` from the second test.